documentation.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit documentation
+class: running-cockpit body-cockpit
+---
+
+## Installation
+
+To get Cockpit installed and running on your system, follow the instructions on the [installation and running page](/running.html).
+
+After Cockpit is running, visit **ip-address-of-machine:9090** in your web browser. If your browser is on the same machine where Cockpit is running, then visit [localhost:9090](http://localhost:9090/) (for example).
+
+## Usage
+
+[The Red Hat Customer Portal product documentation pages](https://access.redhat.com/pilot-documentation/red_hat_enterprise_linux/8) document how to use the "web console" (another name for Cockpit). While this is specifically for the version of Cockpit that ships in Red Hat Enterprise Linux, the interface and its usage is similar on other distributions where the Cockpit web console exists.
+
+- [Getting started using the RHEL web console](https://access.redhat.com/documentation/en-us/red-hat-enterprise-linux/8/guide/8c44950e-29f0-4150-89a5-036f57bf775e)
+- [Managing user accounts in the web console](https://access.redhat.com/documentation/en-us/red-hat-enterprise-linux/8/guide/715670f6-0eb9-4175-a36f-445acd1b1b25)
+- [Managing services in the web console](https://access.redhat.com/documentation/en-us/red-hat-enterprise-linux/8/guide/032603a4-660f-4c0b-97ba-20d32479998e)
+
+## Cockpit Guide
+
+The [Cockpit guide](/guide/latest/) contains:
```

Comment:
This link does not work on your preview, as it drops the /cockpit-project.github.io/ path prefix. Possibly writing this as `./guide/latest/` might work? But anyway, this PR does not change the guide itself, and it should work on the official page.

Reply:
Oh drat. I goofed. It'll work on the main site, but I should fix this. Even though it doesn't matter for the main site, it does matter for our previews.

Reply:
```suggestion
The [Cockpit guide]({{ site.baseurl }}/guide/latest/) contains:
```

I could also remove the / too, but this is more correct.